Accessibility/Accessibility/AppDelegate.swift

```diff
@@ -11,11 +11,19 @@ import UIKit
 @UIApplicationMain
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
-    var window: UIWindow?
+    var window:UIWindow? = UIWindow(frame: UIScreen.main.bounds)
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        
+        let mainVC = HomeShopVC()
+        let navController = UINavigationController(rootViewController: mainVC)
+        
+        //self.window? = UIWindow(frame: UIScreen.main.bounds)
```

Comment:
I actually like the `window` property being assigned here rather than on L:14

Reply:
fixed